### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_AllJobs_Keyboard:"All jobs" arrow is not accesible through keyboard and the focus is not logical .

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5744]
**Bug Description:**
Check out Keros! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
 GitHubTags: #A11yMAS; #A11yTCS; #MAS2.1.1;MAS2.4.3 #A11y_AzureDataStudio; #Win32; #DesktopApp
**Environment Details:**
Application name: Azure Data Studio (Agent, Create External Data, Notebooks)
App Version: 1.8.0-insider (user setup)
VS Code 1.33.1
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.18362
**Additional Details:**
Tool Name:
MAS Reference:MAS2.4.3,2.1.1
**Repro Steps:**
1.Install the app Azure Data Studio.
2.Naviagte to the SQL agent tab->Jobs->Click on any the jobs in the table.
3.observed the key board focus on arrow near "all jobs" heading.
**Actual:**
Issue 1.Key board focus is not logical as it directly jump on run button as soon as we enter in all jobs page and also that arrow is not accessible through keyboard.
Issue2.After "Refresh" tab focus directly jump on tree view escaping over view tab.

![image](https://user-images.githubusercontent.com/39204909/58628127-9121fc00-82f6-11e9-943f-c00e540f3ed2.png)

**Expected:**
Issue 1."all jobs" arrow should be accessible through keyboard and the first focus should be on that arrow as soon as we enter in that page.
Issue 2. Focus should move on overview tab after refresh tab.
**User Impact:**
User with motor disabilities will find difficult to navigate the page.
Recommendation:
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
**MAS Reference:2.4.3.2.1.1**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}
**Attachment for Reference:**
[5744_AccessibilityA11y_AzureDataStudio_AllJobs_Keyboard.pptx](https://github.com/microsoft/azuredatastudio/files/3236707/5744_AccessibilityA11y_AzureDataStudio_AllJobs_Keyboard.pptx)
Does this issue occur when all extensions are disabled?: Yes

**Comment by AccessibilityTestingTeam-TCS** [comment]
**Verified in the below environment:**
Application name: Azure Data Studio
Version: 1.9.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8:6.6.346.32
OS : OS: Windows_NT x64 10.0.18362
Tool Name: Keyboard
Bug has not been fixed it is still repro hence reopening the bug.
**Attachment for reference:-**		

[5744_Reopened_A11y_AzureDataStudio_Alljobs_Screenreader.pptx](https://github.com/microsoft/azuredatastudio/files/3285542/5744_Reopened_A11y_AzureDataStudio_Alljobs_Screenreader.pptx)
